packages/teams-js/src/public/storage.ts

```diff
@@ -0,0 +1,34 @@
+import { GlobalVars } from '../internal/globalVars';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { HostClientType } from './constants';
+import { runtime } from './runtime';
+
+/**
+ * Contains functionality to allow web apps to store data in webview cache
+ *
+ * @beta
+ */
+export namespace storage {
+  /**
+   * Checks if web storage gets cleared when a user logs out from host client
+   *
+   * @returns true is web storage gets cleared on logout and false if it does not
+   */
+  export function isWebStorageClearedOnUserLogOut(): boolean {
```

Comment:
Something that came up in a different PR -- can you please add the @beta tag to all functions/interfaces/enums, etc. defined in this file? This is so the purple beta text appears correctly in the public documentation.

Reply:
sure,  is this needed for isSupported function also ?